README.md

```diff
@@ -1,13 +1,149 @@
-# connecthub
+# ConnectHub
 
-# Run the web version locally
+***
 
-This project is unlikely to be deployed, everything will happen locally. To run the website:
+## Overview
+ConnectHub is a forum for meaningful discussions, offering easy post management and personalized feeds based on your interests.
 
-1. `npm install` to install all front end dependencies (make sure you have `npm` installed, this project uses the newest node version `v20.17.0`)
-2. Load the `pom.xml` to install all back end dependencies
-3. `npm run watch` to build all static assets
-4. Run the `WebServer.java` script in `src/main/java/app`
-4. navigate to `http://localhost:8080/signup` to start (sign up first!)
+## Table of Contents
+1. [Description](#description)
```

Comment:
according to the checklist we need links to all the sections in the readme : )

Reply:
oops forgot about this :)